operator/config/rbac/auth_proxy_role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: proxy-role
+rules:
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - create
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
```

Comment:
Do you know why it requires `subjectaccessreviews` without any scope?

Reply:
Please see this comment
https://github.com/gravitational/teleport/pull/13331#issuecomment-1161460367